packages/hooks/src/useWalletAccountsList.ts

```diff
@@ -0,0 +1,76 @@
+import { useMemo } from 'react';
+import useAccountsList from './useAccountsList';
+
+const currency_to_icon_mapper: Record<string, string> = {
```

Comment:
@sergei-deriv Suggestion, Let's keep the `dark` and `light` icons, Same for the gradient 🙏🏻 The decision on removing dark mode doesn't make any sense and definitely they gonna ask us to add dark mode again, So we can already avoid making more work for ourselves, Just leave it here then in the component level we just don't use them for now.